### PR TITLE
✅ Test the lesson page contains the Markdown

### PR DESCRIPTION
**Pull request by kylewelsby**
### Context

Ticket: CMB-578

### Changes proposed in this pull request

Adds Cypress testing for Lesson pages

### Guidance to review

```bash
git checkout testing/lessons
npm install
npm test:e2e
```

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -107,3 +114,34 @@ Cypress.Commands.add("login", () => {
     }
   });
 });
+
+Cypress.Commands.add("shouldContainMarkdown", (selector, markdownStringRaw) => {
```
Comment (rjlynch) [inline cypress/support/commands.js:118]: neat

File: cypress/integration/Lesson.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+describe("Lesson", () => {
+  beforeEach(() => {
+    cy.loadLessonFixtures();
+    cy.login();
+  });
+
+  it("Displays lesson contents", () => {
+    cy.get("@lessons").each(lesson => {
+      cy.log(lesson);
```
Comment (rjlynch) [inline cypress/integration/Lesson.spec.js]: is this a debug line or do we need this? Not super familiar with cypress

**Review by rjlynch — APPROVED** [review]
Nice 👍

File: cypress/integration/Lesson.spec.js (hunk 2 above)
Comment (kylewelsby) [reply cypress/integration/Lesson.spec.js]: No, it's not,  It can go.
```suggestion
```

**Pull request merged**